actionview/test/template/dependency_tracker_test.rb

```diff
@@ -48,69 +55,72 @@ def test_returns_empty_array_if_no_tracker_is_found
   end
 end
 
-class ERBTrackerTest < Minitest::Test
+class RipperTrackerTest < Minitest::Test
   def make_tracker(name, template)
-    ActionView::DependencyTracker::ERBTracker.new(name, template)
+    ActionView::DependencyTracker::RipperTracker.new(name, template)
   end
 
   def test_dependency_of_erb_template_with_number_in_filename
-    template = FakeTemplate.new("<%# render 'messages/message123' %>", :erb)
+    template = FakeTemplate.new("<%= render 'messages/message123' %>", :erb)
```

Comment:
Was there a reason these changed? 🤔

Reply:
Yes! we correctly skip commented out render calls now! 🎉  so to make this test pass it has to be a real render call.